src/test/components/__snapshots__/LocalTrack.test.js.snap

```diff
@@ -252,8 +252,6 @@ exports[`timeline/LocalTrack with an IPC track matches the snapshot of the IPC t
             <div>
               <canvas
                 class="timelineMarkersCanvas"
-                height="400"
-                width="400"
```

Comment:
Not sure why we don't have these anymore, but they are still not changed.

Reply:
I believe you need to call `flushRafCalls();` after dispatching `showLocalTrack`.